pymongo/uri_parser.py

```diff
@@ -129,50 +130,33 @@ def parse_host(entity, default_port=DEFAULT_PORT):
     return host.lower(), port
 
 
-def validate_options(opts, warn=False):
```

Comment:
We can't remove this method so suddenly since it's a public api.

Reply:
Understood. I didn't think about the fact that this is all public API - I'll add it back. I just don't understand why we have a method that does nothing but wrap `common.get_validated_options`.